src/statuspage.js

```diff
@@ -19,24 +19,15 @@ const request = require('request-promise-native');
 const status = 'operational';
 
 let logger = console;
+const config = {};
 let packageJSON = {};
-let defaultName;
-let defaultDescription;
-let defaultGroup;
 
 try {
   packageJSON = JSON.parse(fs.readFileSync('package.json'));
-  if (packageJSON.statuspage) {
-    defaultName = packageJSON.statuspage.name || packageJSON.name || undefined;
-    defaultDescription = packageJSON.statuspage.description || packageJSON.description || undefined;
-    defaultGroup = packageJSON.statuspage.group || undefined;
-  } else {
-    defaultName = packageJSON.name;
-    defaultDescription = packageJSON.description;
-    defaultGroup = undefined;
-  }
+  config.name = packageJSON.name;
+  config.description = packageJSON.description;
 } catch (e) {
-  defaultName = undefined;
+  config.name = undefined;
```

Comment:
why this?

Reply:
Because eslint would complain about an empty `catch` block :)